server/middlewares/csp.js

```diff
@@ -4,7 +4,7 @@ const directives = {
   'connect-src': ['*'],
   'img-src': ["'self'", 'data:'],
   'script-src': [
-    "'self'",
+    "'self' 'unsafe-eval'",
```

Comment:
is unsafe-eval still required?

Reply:
I'm afraid yes, without the unsafe-eval the wasm streaming initiation fails and I havent been able to find any other solution to this.